relayer/relayer.go

```diff
@@ -105,9 +107,19 @@ func NewRelayer(
 		logger:                   logger,
 	}
 
-	// Start the message router. We must do this before Subscribing for the first time, otherwise we may miss an incoming message
+	// Start the message router. We must do this before Subscribing or Initializing for the first time, otherwise we may miss an incoming message
 	go r.RouteToMessageChannel()
 
+	// Initialize the subscriber. This will poll the node for any logs that match the filter query from the stored block height,
```

Comment:
I think "query" is more accurate rather than "poll" in this case.

Reply:
Updated.